Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# 22nd Small Improvments

## Changelog:
- Evil characters should have less default personal traits.
- Independent AI rulers are less likely to do evil actions like converting to evil religions: Sargerite, Death God, etc.


## Scripting changelog (ignore if you can't read CK2 script):
- Refactored form_the_empire_of_zul, its code style is more close to form_the_gorian_empire now.
- liberator_of_lordaeron, scarlet_king, emperor_of_goria allow bastards.
- Warrior traits no longer give giant `ai_ambition`.
- Paladin traits no longer give giant `ai_zeal`.


# How to test:
Start the game.
Check evil characters.
Play for some years, there should be less independent rulers converting.
